Maven release fails during javadoc generation

```
[INFO] [ERROR] error: module okhttp3.urlconnection reads package okhttp3.internal from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR] error: module okio reads package okhttp3 from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR] error: module okio reads package okhttp3.internal from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR] error: module org.kohsuke.github.api reads package okhttp3 from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR] error: module org.kohsuke.github.api reads package okhttp3.internal from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR] error: module okhttp3 reads package okhttp3 from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR] error: module okhttp3 reads package okhttp3.internal from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR] error: module com.github.spotbugs.annotations reads package okhttp3 from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR] error: module com.github.spotbugs.annotations reads package okhttp3.internal from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR] error: module org.apache.commons.lang3 reads package okhttp3 from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR] error: module org.apache.commons.lang3 reads package okhttp3.internal from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR] error: module org.apache.commons.io reads package okhttp3 from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR] error: module org.apache.commons.io reads package okhttp3.internal from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR] error: the unnamed module reads package okhttp3 from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR] error: the unnamed module reads package okhttp3.internal from both okhttp3 and okhttp3.urlconnection
[INFO] [ERROR]
```

--- a/pom.xml
+++ b/pom.xml
@@ -215,7 +215,7 @@
           <artifactId>maven-javadoc-plugin</artifactId>
           <version>3.4.1</version>
           <configuration>
-            <source>11</source>
+            <release>8</release>
             <failOnWarnings>true</failOnWarnings>
             <doclint>all</doclint>
           </configuration>

